.changeset/mighty-pugs-knock.md

```diff
@@ -0,0 +1,7 @@
+---
+'@clerk/clerk-sdk-node': major
+'@clerk/backend': major
+'@clerk/nextjs': major
+---
+
+Change the response payload of Backend API requests to return `{ data, errors }` instead of return the data and throwing on error response.
```

Comment:
We should give people a Before/After example to show how they can migrate

Reply:
I have provided an example of how to keep the same behavior.